### DISCUSSION TITLE
Hello sir

### DISCUSSION THREAD

**Discussion by komal-2022** [Jay-study-nildana/FrontEndForStudents#30] [General]
@Jay-study-nildana I am not able to clone in git bash getting this error

KP@kompy MINGW64 ~/Videos/Edureka
$  git clone https://github.com/komal-2022/helloworldJun16.git
Cloning into 'helloworldJun16'...
****remote: Repository not found.
fatal: repository 'https://github.com/komal-2022/helloworldJun16.git/' not found****

**Comment by Jay-study-nildana** [comment]
@komal-2022 

welcome to my learning discussion forum. 

for github, first go to this link 

[Setting Up GitHub VS Code and FireFox](https://github.com/Jay-study-nildana/FrontEndForStudents/discussions/15)

and if you still have more questions, reply in this forum. 

Thanks

(and don't forget to follow me on linked in, twitter and youtube and all those places. all these links are available on my github account)

watch this video for more details : https://youtube.com/shorts/9PkFfAQ3Jt0

Happy Learning.

Jay
